test/SILOptimizer/mandatory_inlining_devirt.swift

```diff
@@ -30,3 +30,18 @@ public class PublicClass {
   }
 }
 
+public protocol Thrower {
```

Comment:
This test needs FileCheck patterns.

Reply:
It has them, see the function `caller()`?

Reply:
I don't know what you are talking about. I see caller on line 45 without a FileCheck pattern showing that the callee was or was not devirtualized. Or am I misreading the test?

Reply:
You're right, fixed